src/core/api/api.py

```diff
@@ -611,3 +611,43 @@ async def get_object_image(
     return StreamingResponse(
         _stream(gen), media_type="multipart/x-mixed-replace;boundary=frame"
     )
+
+
+@core_api.get(
+    "/projects/{project_id}/jobs/{job_id}/objects",
+    response_model=Dict[str, Any],
+)
+def get_objects_from_job(
+    project_id: int = Path(..., ge=1),
+    job_id: int = Path(..., ge=1),
+    start: int = Query(0, ge=0),
+    lenght: int = Query(10, ge=1),
```

Comment:
```suggestion
    length: int = Query(10, ge=1),
```

Reply:
This PR is based upon #72 and this is from there. It is all ready fixed in that PR.